source/troubleshooting_webforms.md

```diff
@@ -12,3 +12,7 @@ iOS (running on iPhones and iPads) devices have several known limitations with u
 * Version 8.x shows "undefined is not an object (evaluating 'c.resources')". Solution: Update to latest iOS version
 
 If data collection offline is not required and you are seeing an error on iOS, consider using the [online-only version](https://ee.kobotoolbox.org/::ABCD) instead of the offline URL.
+
+## Data loss
+
+When collecting data through Enekto, users should never clear the cache of the browser (either manually or using some app). Clearing the cache of your browser (during data collection) will clear all the information that is stored in the browser and thus your information will not reach your KoBoToolbox server. This data loss is irreversible. Hence we strongly advise users to clear the cache of the browser if and only if you have successfully submitted all your data to your KoBoToolbox server.
```

Comment:
Maybe try rephrasing this a little 🙂 here's a suggestion:

When collecting data through Enekto, users should never clear the browser's cache (either manually or using some app). Clearing your browser's cache (during data collection) will remove all the information stored in the browser and thus your information will not reach your KoBoToolbox server. This data loss is irreversible. Hence we strongly advise users to clear the browser's cache if and only if you have successfully submitted all your data to your KoBoToolbox server.